backend/fregepoc/repositories/tasks.py

```diff
@@ -37,15 +37,16 @@ def init_worker(**kwargs):
         return
 
     for indexer_cls in indexers:
-        if indexer_cls.__name__ != "SourceforgeIndexer": # Temporarily disable SourceforgeIndexer
-            crawl_repos_task.apply_async(args=(indexer_cls.__name__,))
+        crawl_repos_task.apply_async(args=(indexer_cls.__name__,))
 
 
 @shared_task
 def crawl_repos_task(indexer_class_name):
     indexer_model = apps.get_model("indexers", indexer_class_name)
     indexer: BaseIndexer = indexer_model.load()
-    for repo in indexer:
+
+    batch = next(iter(indexer))
```

Comment:
Here I would rather give a default parameter: `batch = next(iter(indexer), [])`. This will prevent from `StopIteration` error being raised and not handled.